[2024-11-05T20:25:35.287-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_with_custom_email_alerts1.task_merge_participant_status_and_demographics scheduled__2024-11-05T01:24:22.543303+00:00 [queued]>
[2024-11-05T20:25:35.294-0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_with_custom_email_alerts1.task_merge_participant_status_and_demographics scheduled__2024-11-05T01:24:22.543303+00:00 [queued]>
[2024-11-05T20:25:35.294-0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-11-05T20:25:35.430-0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): task_merge_participant_status_and_demographics> on 2024-11-05 01:24:22.543303+00:00
[2024-11-05T20:25:35.435-0500] {standard_task_runner.py:57} INFO - Started process 55534 to run task
[2024-11-05T20:25:35.439-0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline_with_custom_email_alerts1', 'task_merge_participant_status_and_demographics', 'scheduled__2024-11-05T01:24:22.543303+00:00', '--job-id', '351', '--raw', '--subdir', 'DAGS_FOLDER/try1.py', '--cfg-path', '/tmp/tmpwmplb9oh']
[2024-11-05T20:25:35.441-0500] {standard_task_runner.py:85} INFO - Job 351: Subtask task_merge_participant_status_and_demographics
[2024-11-05T20:25:35.611-0500] {task_command.py:416} INFO - Running <TaskInstance: data_pipeline_with_custom_email_alerts1.task_merge_participant_status_and_demographics scheduled__2024-11-05T01:24:22.543303+00:00 [running]> on host mrudula-OMEN-by-HP-Laptop
[2024-11-05T20:25:35.960-0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline_with_custom_email_alerts1' AIRFLOW_CTX_TASK_ID='task_merge_participant_status_and_demographics' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T01:24:22.543303+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-05T01:24:22.543303+00:00'
[2024-11-05T20:25:36.223-0500] {python.py:194} INFO - Done. Returned value was:       Participant_ID  COHORT ENROLL_DATE  ... RANOS  RAWHITE  RAUNKNOWN
0               3000       2  2011-02-01  ...   0.0      1.0          0
1               3001       1  2011-03-01  ...   0.0      1.0          0
2               3002       1  2011-03-01  ...   0.0      1.0          0
3               3003       1  2011-04-01  ...   0.0      1.0          0
4               3004       2  2011-04-01  ...   0.0      1.0          0
...              ...     ...         ...  ...   ...      ...        ...
4603          381012       1  2024-10-01  ...   0.0      1.0          0
4605          381136       4  2024-09-01  ...   0.0      1.0          0
4610          386466       4  2024-09-01  ...   0.0      1.0          0
4611          388122       4  2024-10-01  ...   0.0      1.0          0
4618          402314       1  2024-10-01  ...   0.0      1.0          0

[3737 rows x 25 columns]
[2024-11-05T20:25:36.403-0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_pipeline_with_custom_email_alerts1, task_id=task_merge_participant_status_and_demographics, execution_date=20241105T012422, start_date=20241106T012535, end_date=20241106T012536
[2024-11-05T20:25:36.534-0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-05T20:25:36.556-0500] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
